jwst/datamodels/schemas/fgs.schema.yaml

```diff
@@ -0,0 +1,132 @@
+1allOf:
```

Comment:
The character "1" in the first column appears to be a typo. Should not be there.